wsgi_kerberos.py

```diff
@@ -23,7 +23,7 @@ def ensure_bytestring(s):
     return s.encode('utf-8') if isinstance(s, unicode) else s
 
 
-def _consume_request(environ):
+def _consume_request(environ, chunklen=1024, read_max=1024*1024):
```

Comment:
Looks like golang uses a read max of 2 Kib (weirdly written as `2 << 10`):
https://github.com/golang/go/blob/ee3dded36d69264998c39af0ec851371850d842b/src/net/http/client.go#L616-L625
Can reduce this default to match that in my next round of addressing review feedback.

Reply:
Done in the latest revision.